internal/kafka/internal/services/kafkatlscertmgmt/kafka_tls_certificate_management_service.go

```diff
@@ -218,6 +218,7 @@ func createCertMagicClient(awsConfig *config.AWSConfig,
 		AccountKeyPEM:           kafkaTLSCertificateManagementConfig.AutomaticCertificateManagementConfig.AcmeIssuerAccountKey,
 		DNS01Solver: &certmagic.DNS01Solver{
 			DNSProvider: provider,
+			TTL:         awsConfig.Route53.RecordTTL,
```

Comment:
@miguelsorianod previously, we were not setting the TTL. The default of 0 was then used.
This ensures that the ttl is set when the temporary txt record is created.

Reply:
Why is having a TTL for the temporary txt record good / wanted?

Reply:
Only seeing this now, sorry for the delay.
The idea is that is to leverage cache on any intermediate DNS servers (if any) when the DNS challenge is being resolved: for the check on the client side and once we ask Letsencrypt to resolve the challenge.